aws/cloudformation/bootstrap_chef_stack.sh.erb

```diff
@@ -121,6 +121,7 @@ cat <<JSON > $FIRST_BOOT
     "override_dashboard": "<%=studio_subdomain%>",
     "override_pegasus": "<%=subdomain%>",
 <% end -%>
+    "pd_workshop_queue_url": "https://sqs.$REGION.amazonaws.com/${AWS::AccountId}/pd_workshop-$ENVIRONMENT"
```

Comment:
FYI @breville this change interacts with https://github.com/code-dot-org/code-dot-org/pull/29936.

Reply:
good call-out, I'll be sure to coordinate on this. Getting this config-property removed entirely would simplify things and reduce the risk involved in this change, so I'm inclined to wait until the removal gets merged first.